site/content/en/docs/manual/advanced/projects.md

```diff
@@ -2,17 +2,112 @@
 title: 'Projects page'
 linkTitle: 'Projects page'
 weight: 1
-description: 'Creating and exporting projects in CVAT.'
+description: 'Projects documentation provides details on how to create the CVAT project, load the CVAT project from a backup, and navigate through the project.'
+
+
 ---
 
 ## Projects page
 
-On this page you can create a new project, create a project from a backup, and also see the created projects.
+CVAT project (hereinafter - project) is a top-level container that helps you to keep CVAT tasks of the same type better organized.
+A project usually corresponds to a dataset.
+
+Use this page to create a Project with specified settings like issue tracker, source storage, target storage, bug tracker, segment size, image quality, and others. Settings of the project will further propagate to the task(s) associated with it.
+
+To open the Project page:
+
+1. [Log in](https://app.cvat.ai/) to your CVAT account.
+2. On the top navigation bar, click **Projects**.
+
+You will see the following page:
+
+
+![CVAT Project main page](../../../images/cvat-project-main-page.png)
```

Comment:
The link doesn't work. Please read how to run documentation server locally (`hugo server`)